src/__tests__/posthog-persistence.js

```diff
@@ -1,8 +1,12 @@
 import { PostHogPersistence } from '../posthog-persistence'
 
-given('lib', () => new PostHogPersistence({ name: 'bla', persistence: 'cookie' }))
+let i = 0
 
 describe('persistence', () => {
+    beforeEach(() => {
+        given('lib', () => new PostHogPersistence({ name: 'bla', persistence: 'cookie', token: `${i++}` }))
```

Comment:
Hmm. I see this is needed for cache busting.

Suggestion: Instead of doing it this way, I recommend dropping the cache after every test. This avoids issues where another test starts breaking because of this and avoids this weirdness.

```
afterEach(() => {
  given.lib.clear()
})
```